lib/hanami/environment.rb

```diff
@@ -461,7 +459,7 @@ def set_hanami_env_vars!
     # @since 0.2.0
     # @api private
     def set_application_env_vars!
-      Dotenv.load     root.join(DEFAULT_DOTENV)
+      return unless defined?(Dotenv)
       Dotenv.overload root.join(DEFAULT_DOTENV_ENV % environment)
```

Comment:
@aderyabin Because now we're calling Dotenv just once, shall we use `#load` instead?